docs/openstack/backend_services_deployment.md

```diff
@@ -62,6 +62,25 @@ podified OpenStack control plane services.
 
 ## Pre-checks
 
+Get the database_connection and transport_url cells mappings on the source OSP environment, before those get renamed during
+the [databases adoption](mariadb_copy.md). Note those below to compare it later with the renamed post-adoption values.
+
+Firstly, check the templated view stored in the galera database:
+```bash
+podman exec -it galera-bundle-podman-0 mysql -uroot nova_api -e"select uuid,name,transport_url,database_connection,disabled from cell_mappings"  | grep '|'
+| uuid                                 | name    | transport_url                                                              | database_connection                                                                                                                    | disabled |
+| 00000000-0000-0000-0000-000000000000 | cell0   | none:///                                                                   | mysql+pymysql://{username}:{password}@192.168.122.99/nova_cell0?read_default_file=/etc/my.cnf.d/tripleo.cnf&read_default_group=tripleo |        0 |
+| 58036269-e862-4b3d-943d-2dbd0fc2d915 | default | rabbit://{username}:{password}@standalone.ctlplane.localdomain:5672/?ssl=0 | mysql+pymysql://{username}:{password}@192.168.122.99/nova?read_default_file=/etc/my.cnf.d/tripleo.cnf&read_default_group=tripleo       |        0 |
+```
+
+Then note its Nova representation (showing names and passwords):
+```bash
+podman exec -it nova_api nova-manage cell_v2 list_cells --verbose | grep '|'
+|   Name  |                 UUID                 |                                    Transport URL                                     |                                                               Database Connection                                                               | Disabled |
+|  cell0  | 00000000-0000-0000-0000-000000000000 |                                       none:///                                       | mysql+pymysql://nova:4w6xaYHjGSGJDv66z7xhHQioz@192.168.122.99/nova_cell0?read_default_file=/etc/my.cnf.d/tripleo.cnf&read_default_group=tripleo |  False   |
```

Comment:
@SeanMooney could you please confirm the new name for nova_cell0 DB? post adtopion